lib/find_circle_yml/github/configuration.rb

```diff
@@ -33,14 +36,14 @@ def service
         GitHub::Service.new(
           environment.fetch('GITHUB_USER'),
           environment.fetch('GITHUB_ACCESS_TOKEN'),
-          environment.fetch('GITHUB_ORGANIZATION')
+          environment.fetch('GITHUB_ORGANIZATION', nil)
         )
       end
 
       private
 
       def missing
-        ENVIRONMENT_VARIABLES.reject do |environment_variable|
+        REQUIRED_ENVIRONMENT_VARIABLES.reject do |environment_variable|
           environment.key?(environment_variable)
         end
       end
```

Comment:
It may also be prudent to also print a warning here if organization is not provided, as I expect the main use case / consumer here are orgs. Something like `You have not specified a Github Organization, this will search your personal repos, and _not_ an organization.`.